packages/terra-dynamic-grid/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Dynamic Grid
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-dynamic-grid.svg)](https://www.npmjs.org/package/terra-dynamic-grid)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+{insert description}
```

Comment:
For the description, you can add `The terra-dynamic-grid component provides users a way to build dynamic layouts using CSS Grids.`